src/EnergyPlus/HeatBalanceManager.cc

```diff
@@ -5291,6 +5291,7 @@ namespace HeatBalanceManager {
 
 				if ( MaxHeatLoadZone( ZoneNum ) > 1.0e-4 ) { // make sure load big enough to divide
 					MaxHeatLoadZone( ZoneNum ) = std::abs( max( MaxHeatLoadZone( ZoneNum ), MinLoad ) );
+					MaxHeatLoadPrevDay( ZoneNum ) = std::abs( max( MaxHeatLoadPrevDay( ZoneNum ), MinLoad ) );
```

Comment:
It needs to be done before calculating the TestMaxHeatLoadValue test value.  as it is now the the difference can be calculated with inconsistent application of MinLoad in the two terms in the next line.  The reporting for warmup convergence can be wrong even though as Rich says the actual values are okay.